docs/creating-service-bindings/service-binding-resource.md

```diff
@@ -4,8 +4,122 @@ sidebar_position: 6
+| .applications | string | It describes each bound application resource |
+| .conditions | string | It represents the latest available observations of Service Binding's state |
+| .secret | string | represents the name of the binding secret created by the Service Binding Operator |
+
+A sample service binding status:
+
+```yaml
+status:
+  applications:
+  - name: nodejs-app
+    group: apps
+    version: v1
+    resource: deployments
+  conditions:
+  - lastHeartbeatTime: "2020-10-15T13:23:36Z"
+    lastTransitionTime: "2020-10-15T13:23:23Z"
+    status: "True"
+    type: CollectionReady
+  - lastHeartbeatTime: "2020-10-15T13:23:36Z"
+    lastTransitionTime: "2020-10-15T13:23:23Z"
+    status: "True"
+    type: InjectionReady
+  - lastHeartbeatTime: "2020-10-15T13:23:36Z"
+    lastTransitionTime: "2020-10-15T13:23:23Z"
+    status: "True"
+    type: Ready
+  secret: binding-request-72ddc0c540ab3a290e138726940591debf14c581
+```
+
+### Conditions
+
+Service Binding Status Conditions have three types:
+
+| Condition | Description |
+| -------- | ----------- |
+| CollectionReady | It represents collection of secrets from the service. On failure of collecting secrets from the service, the `conditions.status` becomes "False" with a reason.
+ |
+| InjectionReady | It represents injection of secret into the application. On failure to inject information into the service, the `conditions.status` becomes "False" with a reason. |
+| Ready | It represents overall ready status. On failure, the `conditions.status` becomes "False" with a reason. |
+
+Conditions can have the following combination of type, status and reason:
+
+| Type            | Status | Reason               | Type           | Status | Reason                   | Type           | Status | Reason                    |
+| --------------- | ------ | -------------------- | -------------- | ------ | ------------------------ |----------------|--------|---------------------------|
+| CollectionReady | False  | EmptyServiceSelector | InjectionReady | False  |                          | Ready          | False  | EmptyServiceSelector      |
+| CollectionReady | False  | ServiceNotFound      | InjectionReady | False  |                          | Ready          | False  | ServiceNotFound           |
+| CollectionReady | True   |                      | InjectionReady | False  | EmptyApplicationSelector | Ready          | True   | EmptyApplicationSelector  |
+| CollectionReady | True   |                      | InjectionReady | False  | ApplicationNotFound      | Ready          | False  | ApplicationNotFound       |
+| CollectionReady | True   |                      | InjectionReady | True   |                          | Ready          | True   |                           |
+
+## Advanced Options
+
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+| .spec.bindAsFiles | bool | When set as `false` enables injecting gathered bindings as env variables into the application/workload |
```

Comment:
binding as files is not advanced option - it another way to inject bindings and something what the spec is forcing.